config/locales/api_entreprise/fr.yml

```
@@ -469,3 +476,8 @@ fr:
         link_kit: 🚀 Kit de mise en production
         link_migration: 📋 Guide de migration V.2 > V.3
         link_return: Retour à la documentation technique 🛠
+  token_mailer:
+    magic_link:
+      link_description:
+        one: Un lien vers votre jeton d'accès vient d'être généré à votre intention. Vous pouvez accéder à ce jeton via le lien
+        others: Un lien vers vos jetons d'accès vient d'être généré à votre intention. Vous pouvez accéder à ce jeton via le lien
```

```suggestion
        other: Un lien vers vos jetons d'accès vient d'être généré à votre intention. Vous pouvez accéder à ce jeton via le lien
```

cela m'étonne que ça passe les tests ici